mdx-models/src/main/java/com/mx/path/model/mdx/model/account/Transaction.java

```diff
@@ -54,8 +53,6 @@ public class Transaction extends MdxBase<Transaction> {
   private Long postedAt;
   @XmlElement(name = "posted_on")
   private LocalDate postedOn;
-  @XmlElement(name = "running_balance")
-  private BigDecimal runningBalance;
```

Comment:
@geovanne-duarte-mx we should not remove running balance field as it's being used by HW and AFCU